Resources/config/yui.yml

```diff
@@ -1112,3 +1112,34 @@ system:
                 ez-app-extension-admin:
                     requires: ['ez-serversideview', 'ez-serversideviewservice', 'ez-app-extension']
                     path: %ez_platformui.public_dir%/js/apps/extensions/ez-app-extension-admin.js
+                ez-userprofileview:
```

Comment:
`yui.yml` is kind of sorted. Given this is about the navigation hub and you only add new views, those module definitions should be close the navigation hub view definition in this file.